### PR TITLE
fix: GitHub developer url not being clickable

### PR DESCRIPTION
**Pull request by Pinguin2001**

### CODE REVIEW

File: src/lib/components/AppInfo.svelte [hunk 1]
```diff
@@ -84,7 +84,15 @@
           <div class="detail-row">
             <span class="label">developer</span>
             <span class="separator">::</span>
-            <span class="value">{APP_INFO.github}</span>
+            <a
+                href={APP_INFO.github}
```
Comment (Abdenasser) [inline src/lib/components/AppInfo.svelte:88]: let's also change the url to neohtop's repo instead

**Review by Abdenasser — CHANGES_REQUESTED** [review]

File: src/lib/components/AppInfo.svelte (hunk 1 above)
Comment (Pinguin2001) [reply src/lib/components/AppInfo.svelte:88]: Done!

Comment (Abdenasser) [reply src/lib/components/AppInfo.svelte:88]: for next times you can resolve the conversation and re-request review again (would get notified on mobile).. but good job @Pinguin2001

**Review by Abdenasser — APPROVED** [review]
lgtm, thank you!

**Pull request merged**